### PR TITLE
fix for git issue 4226

### PR DESCRIPTION
**Pull request by davidmu1**

### CODE REVIEW

**Comment by davidmu1** [comment]
Hi Laura, I had to move this work to a new PR because of GitHub problems. I have changed the title and H1 to governanceResource: register.

**Comment by davidmu1** [comment]
#sign-off

File: api-reference/beta/api/governanceresource-register.md [hunk 1]
```diff
@@ -1,61 +1,74 @@
 ---
-title: "Register governanceResource"
-description: "Register an unmanaged governanceResource object in PIM."
+title: "governanceResource: register"
+description: "Register a governanceResource object in PIM."
 localization_priority: Normal
+author: davidmu1
+ms.prod: "microsoft-identity-platform"
```
Comment (Lauragra) [inline api-reference/beta/api/governanceresource-register.md]: FYI, this topic is missing the doc_type attribute. doc_type: apiPageType

Comment (davidmu1) [reply api-reference/beta/api/governanceresource-register.md]: Added the doc_type attribute.

**Comment by davidmu1** [comment]
#sign-off

**Pull request merged**